devices/MAX1704x/README.MD

```diff
@@ -0,0 +1,24 @@
+# MAX1704x battery gauge
+Library designed for controlling MAX1704x IC.
+
+### Functions
+The binding supports all funcionalities for MAX1704x family. Implemented based on https://github.com/sparkfun/SparkFun_MAX1704x_Fuel_Gauge_Arduino_Library/blob/main/src/SparkFun_MAX1704x_Fuel_Gauge_Arduino_Library.cpp
+
+### Classes
```

Comment:
In Markdown, like in Word, titles should be used in order of level, it's not good to have a level 3 without level 2. And the missing line after.

```suggestion
## Classes

```